assets/src/edit-story/components/keyboard/index.js

```diff
@@ -127,9 +147,7 @@ export function useGlobalKeyDownEffect(
   callback,
   deps = undefined
 ) {
-  if (!globalRef.current) {
-    globalRef.current = document;
-  }
+  const globalRef = getGlobalRef();
```

Comment:
There is `const globalRef = createRef();` declared above already.

Reply:
Done.